staging/src/k8s.io/kubectl/pkg/cmd/delete/delete_flags.go

```diff
@@ -118,8 +126,13 @@ func (f *DeleteFlags) AddFlags(cmd *cobra.Command) {
 	if f.Force != nil {
 		cmd.Flags().BoolVar(f.Force, "force", *f.Force, "If true, immediately remove resources from API and bypass graceful deletion. Note that immediate deletion of some resources may result in inconsistency or data loss and requires confirmation.")
 	}
-	if f.Cascade != nil {
-		cmd.Flags().BoolVar(f.Cascade, "cascade", *f.Cascade, "If true, cascade the deletion of the resources managed by this resource (e.g. Pods created by a ReplicationController).  Default true.")
+	if f.CascadingStrategy != nil {
+		cmd.Flags().StringVar(
+			f.CascadingStrategy,
+			"cascade",
+			*f.CascadingStrategy,
+			`Must be "background", "orphan", or "foreground". Selects to deletion cascading strategy for the dependents (e.g. Pods created by a ReplicationController). Defaults to background `)
```

Comment:
"Select the deletion ..." and you still have an extra space at the end.

Reply:
Sorry... Done :) Next time I'll be more careful about spelling!